django_x509/tests/test_ca.py

```diff
@@ -13,13 +14,20 @@
 
 Ca = load_model('django_x509', 'Ca')
 Cert = load_model('django_x509', 'Cert')
+User = get_user_model()
 
 
 class TestCa(TestX509Mixin, TestCase):
     """
     tests for Ca model
     """
 
+    def setUp(self):
+        User.objects.create_superuser(
+            username='admin', password='tester', email='admin@admin.com'
+        )
+        self.client.login(username='admin', password='tester')
```

Comment:
why is this needed here?

Reply:
Hi, Previously It was implemented as:- only authenticated user should be able to download the CRL, But there was minor bug in the code, which allowed non authenticated user to download the CRL, and the tests written for it could not catch it. So after fixing it, for authenticating the user, we are required to login a User.

Reply:
Hmmm... isn't it required to be able to download CRL without auth, I mean for the network devices!?
Can you please confirm with existing behavior?

Reply:
https://github.com/openwisp/django-x509/pull/117/files#r643225106

Reply:
this test is not necessary, see my comments in the last reviews.
@atb00ker CRL is needed by software like OpenVPN, see `--crl-verify`in the openvpn manual: https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/.